tests/Controllers/RegisterTest.php

```diff
@@ -12,9 +13,10 @@
 /**
  * @internal
  */
-final class RegisterTest extends CIDatabaseTestCase
+final class RegisterTest extends CIUnitTestCase
```

Comment:
Shouldn't this extend the new DatabaseTestCase?

Reply:
Done.